src/reducers/reducer.js

```diff
@@ -1995,6 +2013,38 @@ export const onSetCreateTokenStatus = (state, { payload }) => ({
     createToken: {
       ...state.reown.createToken,
       status: payload,
```

Comment:
Why did you remove the `retrying: false` from this reducer?

Reply:
What do you mean? I didn't remove it, it didn't exist here, this is just setting the status

Reply:
Here we have the flag https://github.com/HathorNetwork/hathor-wallet-mobile/blob/master/src/reducers/reducer.js#L1998

Reply:
Well, this reducer is just changing the status, so it doesn't make sense to set retrying to false here

We have the `onSetCreateTokenRetryDismiss` reducer above that sets the retrying to false